__tests__/app.test.js

```diff
@@ -100,6 +100,15 @@ describe("GET /api/non-existing-endpoint", () => {
         expect(res.body.msg).toBe('The request path does not exist')
         })
     })
+    test("status 404: responds with 'The request path does not exist' if the endpoint does not exist", () => {
```

Comment:
I noticed that you have another 3 tests for paths that don't exist in this describe block. We only need one, as we're testing the same behaviour each time for a path that doesn't exist, so you can delete 3 of them 👍